src/main/docs/guide/installation/installFromSource.adoc

```diff
@@ -0,0 +1,55 @@
+Clone the repository as follows:
+
+[source,bash]
+----
+$ git clone https://github.com/micronaut-projects/micronaut-starter.git
+----
+
+
+`cd` into the `micronaut-starter` directory and run the following command:
+
+[source,bash]
+----
+$ ./gradlew micronaut-cli:assembleDist
+----
+
+This creates a ZIP of the executables and lib support for use in the CLI, named with the current version.
+
+You can find it in the `starter-cli/build/dist` folder.
```

Comment:
This will create a zip distribution of the CLI in the `starter-cli/build/dist/` folder.